### DISCUSSION TITLE
Time zone change

### DISCUSSION THREAD

**Discussion by fabricejoubert** [j-kaltes/Juggluco#122] [General]
Hello,
When changing time zones and changing phone time, xdrip no longer broadcasts data in “patched free prodcast” mode. In fact, when moving west, you have to wait for the time difference for data to be sent again. Juggluco must consider that data has already been sent to the range. How can we get around this problem?

**Comment by j-kaltes** [comment]
I don’t know what xDrip does, but Juggluco sends the _Patched Libre Broadcast_ every time a new glucose value arrives independent of phone time or timezone. The timestamp of the data is independent of timezone. Juggluco doesn’t remember what is has already been send to xDrip.
Changing timezone will have no effect on the functioning of Juggluco, but changing the time of the phone does. It will make that  timestamp of some of the data is wrong.
The data in Juggluco is represented in GMT and also the timestamp of the Patched Libre Broadcast is in GMT.

**Reply by fabricejoubert** [reply]
Thank you for this very comprehensive response. The problem must be on the xdrip side which is no longer broadcasting to aaps.
